### PR TITLE
Remove our mkdocs search override

### PR DESCRIPTION
**Pull request by ericholscher**
This removes the JSON build on mkdocs, 
as this has been removed from mkdocs,
so we should stop running it.

It will fix the error here: http://readthedocs.org/projects/mkdocs/builds/6503359/
  
It also fixes the search so that it uses mkdocs builtin search, instead of our own broken search for mkdocs. This is a temporary fix, but will at least make search work.

### CODE REVIEW

File: readthedocs/core/static-src/core/js/readthedocs-doc-embed.js [hunk 1]
```diff
@@ -10,6 +10,6 @@ $(document).ready(function () {
     footer.init();
     sphinx.init();
     // grokthedocs.init();
-    mkdocs.init();
+    // mkdocs.init();
```
Comment (humitos) [inline readthedocs/core/static-src/core/js/readthedocs-doc-embed.js]: Since we are not using it we can also comment line 4 (`require`)

**Comment by ericholscher** [comment]
There's already a million tickets about mkdocs search not being great. They removed the json command that we were using for search on our side, so I'm not sure if "proper" search for mkdocs is in the cards.

**Comment by humitos** [comment]
Yes, I understand that but if their own search doesn't work either I think
it's preferable to remove the search completely from mkdocs to avoid
another tons of issues.

El 11 ene. 2018 11:15 a. m., "Eric Holscher" <notifications@github.com>
escribió:

There's already a million tickets about mkdocs search not being great. They
removed the json command that we were using for search on our side, so I'm
not sure if "proper" search for mkdocs is in the cards.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub
<https://github.com/rtfd/readthedocs.org/pull/3496#issuecomment-356977534>,
or mute the thread

**Comment by ericholscher** [comment]
Their own search works fine, thats what this PR is for :)

**Comment by humitos** [comment]
👍

**Pull request merged**